ui/src/app/base/sagas/websockets.ts

```diff
@@ -485,7 +485,9 @@ export function* handleMessage(
             result = response.result;
           }
         }
-        if (error) {
+        // Sometimes the error response is the original payload sent back, which
+        // can be 0 when requesting a model with an id of 0.
+        if (error || error === 0) {
```

Comment:
This would cause an errant `success` message that would break any `getSuccess` reducer that tried to get something with id `0` that didn't exist.